atom/browser/ui/webui/pdf_viewer_handler.cc

```diff
@@ -112,7 +112,7 @@ void PdfViewerHandler::Initialize(const base::ListValue* args) {
     PopulateStreamInfo(stream_info.get(), stream_, original_url_);
     ResolveJavascriptCallback(*callback_id, *stream_info);
   } else {
-    initialize_callback_id_ = callback_id->CreateDeepCopy();
+    initialize_callback_id_ = std::make_unique(callback_id.Clone());
```

Comment:
Should this be `base::Value::ToUniquePtrValue`? I'm not sure I understand the difference :/